### PR TITLE
Implement TGA pool event cog

### PR DESCRIPTION
**Pull request by 7bitlyrus**
# Pull request template
Use this template for all contributions made for the repository.

Please indicate all that apply with a checkmark. To make a check, convert brackets below to `[x]`.

* [x] **Feature implementation**
* [ ] **Bug fix**
* [ ] **Code change/improvement**
* [ ] **String change (i.e. grammar/spelling error or an addition)**

***

## Describe what your pull request does
Implement TGA pool event cog

## Related issues and extra details
Make sure to add required trophies and wait for @easymac to update API.

### CODE REVIEW

**Comment by 7bitlyrus** [comment]
@easymac Please feel free to add your review!

**Review by mralext20 — APPROVED** [review]

File: events/tgapool.py
Comment (MattBSG) [inline events/tgapool.py:30]: Trophies are all manually pulled? User backgrounds are already getting user info from the user collection, so this could be combined with response_check without any additional db queries.

File: events/tgapool.py
Comment (MattBSG) [inline events/tgapool.py:52]: If this is combined with response_check, then this log could be removed or could instead output to #bot-development

**Review by MattBSG — CHANGES_REQUESTED** [review]

File: events/tgapool.py
Comment (7bitlyrus) [reply events/tgapool.py:30]: Manually pulled, yes. I did try and combine the code, but with the extra trophy logic, the code seemed more confusing when both cases were handled at the same time in the same method.

File: events/tgapool.py
Comment (7bitlyrus) [reply events/tgapool.py:52]: Command is expected to only be once a year. All logs are posted in the event staging channel anyway-- this extra logic is just in case the command is run elsewhere.

File: events/tgapool.py
Comment (MattBSG) [reply events/tgapool.py:30]: If that is the intended case, and immediacy to users isn't wanted, then we can do that. I'll let you merge as a confirmation

**Review by MattBSG — APPROVED** [review]

**Comment by 7bitlyrus** [comment]
Yep! Just waiting for @easymac's approval and for him and @FlapSnapple to submit and agree on trophy designs.

**Review by easymac — APPROVED** [review]
👍

**Pull request merged**
